recipe/meta.yaml

```diff
@@ -44,33 +45,34 @@ requirements:
     - python
     - pip
     - cython
-    - arrow-cpp >=8,<9
+    - arrow-cpp >=10,<11
     - pyarrow
     - numpy
     - setuptools >=40.6.0
     - wheel
   run:
     - python
+    - packaging
     - pyarrow
     - {{ pin_compatible('arrow-cpp', max_pin='x') }}
     - asn1crypto >0.24.0,<2.0.0
     - cffi >=1.9,<2.0.0
-    - cryptography >=3.1.0,<37.0.0
+    - cryptography >=3.1.0,<41.0.0
     - oscrypto <2.0.0
-    - pyopenssl >=16.2.0,<23.0.0
+    - pyopenssl >=16.2.0,<24.0.0
     - pycryptodomex >=3.2,!=3.5.0,<4.0.0
     - pyjwt <3.0.0
     - pytz
     - requests <3.0.0
-    - setuptools >34.0.0
+    - importlib-metadata # [py<38]
```

Comment:
There should be two whitespaces before `# [py<38]` in `importlib-metadata  # [py<38]`